modules/preview-react/divider/lib/Divider.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react';
+import {createComponent} from '@workday/canvas-kit-react/common';
+import {createStencil, handleCsProp, CSProps, calc, cssVar} from '@workday/canvas-kit-styling';
+import {system} from '@workday/canvas-tokens-web';
+
+export const dividerStencil = createStencil({
+  vars: {
+    space: cssVar(system.space.x4),
+  },
+  base: ({space}) => {
+    return {
+      display: 'block',
+      height: 1,
+      border: 'none',
+      borderTop: `1px solid ${system.color.border.divider}`,
+      margin: `${calc.divide(space, 2)} 0`,
```

Comment:
It's be hard to guess as a consumer that the margin is being divided, should we just set the margin to what the consumer sets for top and bottom?

Reply:
Yeah, let's see how people feel about it.